version4.go

```diff
@@ -14,6 +14,14 @@ func New() UUID {
 	return Must(NewRandom())
 }
 
+// NewString creates a new random UUID and returns it as a string or panics.
+// NewString is equivalent to the expression
+//
+//    uuid.Must(uuid.NewRandom()).String()
```

Comment:
I think it might be more helpful to say it is the equivalent of `uuid.New().String()`.

Reply:
Done